config.js

```diff
@@ -5,7 +5,22 @@ const CONFIG = {
   [CONST.CONFIG.LOG_LEVEL_TELETEXT_SERVER]: CONST.LOG_LEVEL_VERBOSE,
   [CONST.CONFIG.LOG_LEVEL_TELETEXT_VIEWER]: CONST.LOG_LEVEL_VERBOSE,
 
-  [CONST.CONFIG.BASE_DIR]: '/var/www/',
+  // this is used by `update-service-pages.js` as the place where the
+  // raw source service pages directories (likely SVN repositories)
+  // are located.
+  // (Note: service pages are not actually served by Muttlee from here
+  //  they need additional renaming first, and are then served from
+  //  CONST.CONFIG.SERVICE_PAGES_SERVE_DIR defined below).
+  [CONST.CONFIG.SERVICE_PAGES_DIR]: '/var/www/teletext-services',
+
+  // (Note: this is the root directory for the live service pages,
+  //  within it should be individual service subdirectories
+  //  e.g. /var/www/private/onair/teefax, /var/www/private/onair/d2k,
+  //  etc. matching the id's of the services defined below in
+  //  CONST.CONFIG.SERVICES_AVAILABLE)
+  [CONST.CONFIG.SERVICE_PAGES_SERVE_DIR]: '/var/www/private/onair',
```

Comment:
@peterkvt80 this is the breaking change, as this will likely not match your current setup